src/meta/src/controller/catalog/drop_op.rs

```diff
@@ -198,9 +198,13 @@ impl CatalogController {
                 .count(&txn)
                 .await?;
             if creating != 0 {
-                return Err(MetaError::permission_denied(format!(
-                    "can not drop {creating} creating streaming job, please cancel them firstly"
-                )));
+                if creating == 1 && object_type == ObjectType::Sink {
```

Comment:
Why we can only drop one sink job at a time?

Reply:
This is a workaround to allow dropping creating sink using `DROP SINK` command. I just want to limit the functionality to "DROP creating sink into table". Currently, using "sink into table" requires replacing the downstream table. It's safer to reject "drop cascade" when there are jobs still being created.